spec/models/reviewable_flagged_post_spec.rb

```diff
@@ -273,4 +273,29 @@ def pending_count
 
   end
 
+  describe "#perform_delete_and_agree" do
+    it "notifies the user about the flagged post deletion" do
+      reviewable = Fabricate(:reviewable_flagged_post)
+      reviewable.add_score(
+        moderator, PostActionType.types[:spam],
+        created_at: reviewable.created_at
+      )
+
+      expect do
+        reviewable.perform(moderator, :delete_and_agree)
+      end.to change(Jobs::SendSystemMessage.jobs, :size).by(1)
```

Comment:
We usually prefer checking that the job that was queued is the one we're actually testing (example:  d7f6d37a985a3e6233d4285f7d55c9e14905e552).